packages/applications-service-api/api/openapi.yaml

```diff
@@ -936,7 +936,7 @@ paths:
         - in: query
           name: caseRef
```

Comment:
I missed this while defining and refining but would it make sense to call this query param caseReference.
There are a lot of inconsistencies in the API so would be good to converge to a consistent convention.

Reply:
Thanks Philip, will do this in an additional PR as discussed.